scout/server/blueprints/cases/static/case_images.js

```diff
@@ -0,0 +1,365 @@
+       {name: '17', length: 165, cent_start: 55, cent_length: 8 },
+       {name: '18', length: 158, cent_start: 42, cent_length: 8 },
+       {name: '19', length: 119, cent_start: 60, cent_length: 8 },
+       {name: '20', length: 127, cent_start: 60, cent_length: 8 },
+       {name: '21', length: 95, cent_start: 33, cent_length: 8 },
+       {name: '22', length: 104, cent_start: 38, cent_length: 8 },
+       {name: 'X', length: 312, cent_start: 127, cent_length: 8 },
+       {name: 'Y', length: 107, cent_start: 32, cent_length: 4 }]
+
+
+/**
+ * Iterate case.individuals. If a path to a image directory
+ * is set, get panels on the page and add image content
+ */
+function add_image_to_individual_panel(individuals, prefixes, institute, case_name){
+    for (var i=0; i<individuals.length; i++){
+        console.log("PATHS: "+individuals[i].chromograph_images)
+        if(individuals[i].chromograph_images){
+            draw_tracks(individuals[i], prefixes, institute, case_name)
+        }
+    }
+}
+
+
+
+
+// <svg id="svg_ADM1059A1"> </svg>
+
+
+/**
+ * Draw RHO call pictures -coverage- and UPD pictures -color coded
+ * genome regions- onto the dashboard.
+ */
+function draw_tracks(individual, prefixes, institute, case_name){
+    const CYT_HEIGHT = 50 ;
+    const CYT_WIDTH = 500 ;
+    var svg_element = document.getElementById("svg_" + individual["individual_id"])
+    console.log("DRAW TRACKS");
+    console.log(individual.chromograph_images)
+
+    if (individual.chromograph_images.autozyg != undefined){
+        console.log("print autozyg: "+ individual.chromograph_images.autozyg);
+        var autozyg_imgPath = create_path(institute, case_name, individual, 'autozyg_images')
+        var autozyg_images = make_names("autozyg-");
+    }
+    if (individual.chromograph_images.upd_regions != undefined){
+        console.log("print UPD_REGIONS: "+ individual.chromograph_images.upd_regions);
+        var upd_regions_imgPath = create_path(institute, case_name, individual, 'upd_regions_images')
+        var upd_regions_images = make_names("upd_regions-");
+    }
+    if (individual.chromograph_images.coverage != undefined){
+        console.log("printCOVERAGE: "+individual.chromograph_images.coverage );
+        var coverage_imgPath = create_path(institute, case_name, individual, 'coverage_images')
+        var coverage_images = make_names("coverage-");
+    }
+
+
+    // ideograms always exist
+    var ideo_imgPath = static_path_ideograms(institute, case_name, individual, 'ideaograms')
+    var ideo_images = make_names('')
+
+    console.log("PATHS")
+    console.log("ideo: " + ideo_imgPath)
+    console.log("autozygosity: " + autozyg_imgPath)
+    console.log("regions: " + upd_regions_imgPath)
+    console.log("coverage" + coverage_imgPath)
+
+
+    var autozyg_imgObj = new Image()
+    var upd_regions_imgObj = new Image()
+    var coverage_imgObj = new Image()
+
+    var number_of_columns = $(window).width() < WIDTH_BREAKPOINT? 2:3
```

Comment:
My rather large monitor would not go to three..

Reply:
Ok. Always so hard to test hardware bounds. And Bootstrap size macros aren't up to date with modern resolutions 🤔.

One idea is to redo the picture layout pairing short and long chromosomes. Like this:

```
===-====  =-==
===-=== ==-===
==-=== ===-===
=-= ==-=======
```
(at the moment too lazy to draw a picture, can you make sense of it?)

Reply:
Pushed a fix. Let me know how it works.